ChatSecure/Classes/View Controllers/OTRMessagesGroupViewController.m

```diff
@@ -44,6 +44,11 @@ - (void)setupGroupChat:(NSArray <NSString *>*)buddies account:(OTRAccount *)acco
     [self setThreadKey:self.threadKey collection:[OTRXMPPRoom collection]];
 }
 
+- (nullable OTRAccount *)accountWithTransaction:(nonnull YapDatabaseReadTransaction *)transaction {
```

Comment:
@chrisballinger Yes, it exists but returns nil.

Good thing I wrote a comment in the commit =)

"super will just return nil, since the ThreadKey has not been set when
this is called during setup."